### ISSUE TITLE
File operations from/to pCloud to Crypto Folder

### ISSUE THREAD

**Issue by mathieucarbou** [pCloud/pcloud-sdk-js#16]
Please see this ticket that I have opened: https://github.com/rclone/rclone/issues/4053

I would like to be able to do some file operations from/to encrypted folder.

Use case is that I have plenty of files I would like to move but I do not want to re-upload them.

Also, I would like to do that through rclone, which is using the REST API I think.

**Comment by elanoism** [comment]
Since this involves client side encryption, anything going in and out of "Crypto Folder" (from and to normal folder) requires an upload. Also encryption.

**Comment by mathieucarbou** [comment]
Good point... So that means if rclone was to provide a move operation, it would need download, then upload then remove. This is acceptable even if slower if the user wants that.

The thing is that right now, I don't think this is even possible to do that ?

Rclone sees everything as encrypted. Is there some API somewhere that would allow file i/o operations on encrypted folder, by asking the user for the the password ?

**Comment by aperkaz** [comment]
@mathieucarbou I have the same use case, I want to use r-sync (or similar) to backup pcloud encrypted vaul to my local file system.
Is there a way, perhaps using the sdk API?
